demo/demo/plotly_apps.py

```diff
@@ -316,8 +322,10 @@ def callback_show_timeseries(internal_state_string, state_uid, **kwargs):
 localState = DjangoDash("LocalState",
                         serve_locally=True)
 
-localState.layout = html.Div([html.Img(src=localState.get_asset_url('image_one.png')),
-                              html.Img(src='assets/image_two.png'),
+localState.layout = html.Div([
+                              html.Img(src=localState.get_asset_url(STATIC_URL + 'assets/plotly_apps/image_one.png')),
```

Comment:
The point of the example is to demonstrate correct usage of the static files functionality. I'm not sure what the goal of this change is, but it doesn't seem to be particularly appropriate.